vestido/vestido-storefront/modules/Checkout/CheckoutView.tsx

```diff
@@ -79,15 +86,32 @@ const CheckoutView: React.FC = () => {
 
   const { trigger: createOrderTrigger } = useCreateOrder();
 
-  const totalPrice =
-    cartItems?.data.reduce((total, item) => {
-      return total + item.qty * item.item.price;
-    }, 0) ?? 0;
+  const totalPrice = buyNowItem
+    ? (checkoutItems?.reduce((total, item) => {
+        return total + item.item.price;
+      }, 0) ?? 0)
+    : (cartItems?.reduce((total, item) => {
+        return total + item.qty * item.item.price;
+      }, 0) ?? 0);
```

Comment:
You only have to do this with `checkoutItems`?

Reply:
ok

Reply:
this is done